### PR TITLE
fix(errors): solve ECR and CodeArtifact errors

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Description

Fix the following errors:
- CodeArtifact: `IndexError[113]: list index out of range`
- ECR: `An error occurred (RepositoryNotFoundException) when calling the ListTagsForResource operation`

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prowler/providers/aws/services/codeartifact/codeartifact_service.py [hunk 1]
```diff
@@ -110,15 +110,19 @@ def __list_packages__(self, regional_client):
...
+                            if latest_version_information["versions"]:
+                                latest_version = latest_version_information["versions"][
+                                    0
+                                ]["version"]
+                                latest_origin_type = latest_version_information[
+                                    "versions"
+                                ][0]["origin"]["originType"]
+                                latest_status = latest_version_information["versions"][
+                                    0
+                                ]["status"]
```
Comment (jfagoagas) [inline prowler/providers/aws/services/codeartifact/codeartifact_service.py]: Could you use the `dict.get` here?

File: prowler/providers/aws/services/ecr/ecr_service.py [hunk 2]
```diff
@@ -155,11 +156,21 @@ def __list_tags_for_resource__(self):
...
             for repository in self.repositories:
-                regional_client = self.regional_clients[repository.region]
-                response = regional_client.list_tags_for_resource(
-                    resourceArn=repository.arn
-                )["tags"]
-                repository.tags = response
+                try:
+                    regional_client = self.regional_clients[repository.region]
+                    response = regional_client.list_tags_for_resource(
+                        resourceArn=repository.arn + "1"
```
Comment (jfagoagas) [inline prowler/providers/aws/services/ecr/ecr_service.py]: Why this "1"?

**Review by jfagoagas — CHANGES_REQUESTED** [review]

File: prowler/providers/aws/services/ecr/ecr_service.py (hunk 2 above)
Comment (MrCloudSec) [reply prowler/providers/aws/services/ecr/ecr_service.py]: It was for testing, I'll remove it

File: prowler/providers/aws/services/codeartifact/codeartifact_service.py (hunk 1 above)
Comment (MrCloudSec) [reply prowler/providers/aws/services/codeartifact/codeartifact_service.py]: Done

**Review by jfagoagas — APPROVED** [review]
🔝

**Pull request merged**